redis-pq-demo/docker-compose.yml

```diff
@@ -0,0 +1,117 @@
+version: '3'
+services:
+  caddy:
+    image: caddy
+    hostname: caddy
+    networks:
+      - acapy_default
+    ports: # Uncomment to access caddy outside of containers
+      - 2015:2015
+    #   - 2019:2019
+    volumes:
+      - ./caddy/Caddyfile:/etc/caddy/Caddyfile:z
+    healthcheck:
+      # Port 2019 is an internal Caddy admin port.
+      test: nc -zv localhost:2019 || exit -1
+      interval: 3s
+      timeout: 3s
+      start_period: 10s
+      retries: 5
+
+  ngrok:
+    image: ngrok/ngrok
+    restart: unless-stopped
+    hostname: ngrok
+    depends_on:
+      caddy:
+        condition: service_healthy
+    networks:
+      - acapy_default
+    environment:
+      - NGROK_AUTHTOKEN=${NGROK_AUTHTOKEN}
+    command: http caddy:2015 --log stdout
+    # ports: # Uncomment to access ngrok outside of containers
+    #   - 4040:4040 # admin port
+    healthcheck:
+      test: /bin/bash -c "</dev/tcp/ngrok/4040"
+      interval: 3s
+      timeout: 3s
+      start_period: 5s
+      retries: 5
+      
+
+  db:
+    image: postgres:15
+    # restart: always
+    environment:
+      POSTGRES_PASSWORD: ${POSTGRESQL_ADMIN_PASSWORD:-development}
+    # ports: # Uncomment to access postgres outside of containers
+    # - "5432:5432"
+    networks:
+      - acapy_default
+    volumes:
+      - agency-wallet:/var/lib/pgsql/data:z
+    healthcheck:
+      test: pg_isready -U postgres
+      interval: 3s
+      timeout: 3s
+      retries: 5
+
+  #*************************************************************
+  # acapy-redis_queue: aca-py that produces redis events. *
+  #*************************************************************
+
+  mediator:
+    build:
+      context: .
```

Comment:
The build for this works (as in it stands up a mediator with redis_queu), but doesn't use ngrok even if you want it.

Reply:
I'm working to reinstate ngrok easily. I agree this should be part of the demo.